gateleen-scheduler/src/main/java/org/swisspush/gateleen/scheduler/SchedulerResourceManager.java

```diff
@@ -177,4 +183,51 @@ public void refresh() {
     public void enableResourceLogging(boolean resourceLoggingEnabled) {
         this.logConfigurationResourceChanges = resourceLoggingEnabled;
     }
+
+
+    /**
+     * Evaluate if the given date is in daylight saving time or not.
+     *
+     * @return true if in daylight saving time, false otherwise
+     */
+    public static boolean isDaylightSavingTimeState(ZonedDateTime checkDate) {
+        return checkDate.getZone().getRules().isDaylightSavings(checkDate.toInstant());
+    }
+
+    /**
+     * Evaluate if the current date is in daylight saving time or not.
+     *
+     * @return true if in daylight saving time, false otherwise
+     */
+    public static boolean isCurrentDaylightSavingTimeState() {
+        return isDaylightSavingTimeState(ZonedDateTime.now());
+    }
+
+    /**
+     * Schedule a periodic check for daylight saving time changes.
+     */
+    private void scheduleDaylightSavingTimeStateChangeObserver() {
+        if (properties != null && properties.get("dst.observe") != null ? (Boolean) properties.get("dst.observe") : false) {
```

Comment:
Use `Boolean.parseBoolean(properties.getProperty("dst.observe"))` instead of `(Boolean) properties.get(...)`

Reply:
ok, but then I will have to cast to String because the Property map contains not strings but Objects (not invented by me, it was already like this).

Reply:
This escalated really bad ....
As the given properties are of Object type, it could be strings or Boolean or whatever. Added a UtilClass to make this hopefully easier to manage for others as well.  It accepts Strings and Booleans.